statistics/update.go

```diff
@@ -179,7 +179,7 @@ type QueryFeedback struct {
 	colID       int64
 	isIndex     bool
 	idxRanges   []*ranger.NewRange
-	intRanges   []ranger.IntColumnRange
+	intRanges   []*ranger.NewRange
```

Comment:
Only one of the `intRanges` and `idxRanges` will be set, so just keep one is fine.